feature-pack/src/main/resources/modules/system/layers/base/org/jgroups/kubernetes/main/module.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2018, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.7" name="org.jgroups.kubernetes">
```

Comment:
My 2 cents: unless there's a reason for making this public, I suggest making this module unsupported, like we do with azure ping.

Reply:
No, we are not making `org.jgroups.azure` unsupported, see: https://github.com/wildfly/wildfly/blob/master/feature-pack/src/main/resources/modules/system/layers/base/org/jgroups/azure/main/module.xml#L25

Reply:
Oh, ok, that's what I get for checking EAP not WildFly. Apparently, it's only unsupported there: https://github.com/jbossas/jboss-eap7/blob/7.1.x/feature-pack/src/main/resources/modules/system/layers/base/org/jgroups/azure/main/module.xml#L28

Nothing then.